robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,112 @@
+from itertools import tee
+
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+def eol():
+    return Token(Token.EOL)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120):
+        self.line_length = line_length
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(token.end_col_offset <= self.line_length for token in node.tokens[::-1]):
```

Comment:
We could save few ms if we were to iterate through lines and check only last token:
```
for line in node.lines:
    if line[-1].end_col_offset > self.line_length:
        break
else:
    return node
```
or
```
if all(line.end_col_offset > self.line_length for line in node.lines):
    return node
```
(I learned about it recently, it automatically splits on ... new lines markers)